sunpy/util/cond_dispatch.py

```diff
@@ -78,7 +78,7 @@
 
 from itertools import chain, repeat
 
-from sunpy.extern.six.moves import zip as izip
+from sunpy.extern.six.moves import zip
```

Comment:
What is wrong with from sunpy.extern import six and just replacing the izip and imap commands?